tools/appimage/build.sh

```diff
@@ -0,0 +1,207 @@
+#! /bin/bash
+#
+# build.sh
+# Builds Randovania into an AppImage
+#
+# This script is mostly built on this guide from AppImageKit:
+# https://github.com/AppImage/AppImageKit/wiki/Bundling-Python-apps
+#
+# Copyright (C) 2022 Salt <jacob@babor.tech>
+# Distributed under terms of the GNU General Public License, Version 3
+#
+
+set -e
+
+#
+# Configuration vars
+# These variables can be freely modified
+#
+
+# The URL to the Python AppImage we'll bundle RDV in
+python_appimage="https://github.com/niess/python-appimage/releases/download/python3.9/python3.9.10-cp39-cp39-manylinux1_x86_64.AppImage"
```

Comment:
Just pushed a few commits that fix this. We also remove our dependency on that Python appimage since we're shipping a bundled python with the release image